contracts/levels/Elevator.sol

```diff
@@ -14,8 +14,8 @@ contract Elevator {
     Building building = Building(msg.sender);
 
     if (! building.isLastFloor(_floor)) {
+      top = building.isLastFloor(_floor);
       floor = _floor;
-      top = building.isLastFloor(floor);

```

Comment:
Why is this file from a different level being modified?

Reply:
Because this level should be done only expected way. Modifying prevents solution from new shop level.

Reply:
I understand, but keep in mind that the source code displayed by Ethernaut must match whatever is deployed. A modification to a level should be a separate PR, and it should be considered carefully since it would mean that history is lost for all players who have beat the level. This will be solved in the future when levels are upgradeable using ZeppelinOS. For now, lets keep this PR specific to the Shop level :pray: